cmd/wallet-web/src/mixins/gnap/gnap.js

```diff
@@ -15,16 +21,27 @@ export async function gnapRequestAccess(signer, gnapAccessTokens, gnapAuthServer
       start: ['redirect'],
       finish: {
         method: 'redirect',
-        uri: '/',
+        uri: walletWebURL + '/gnap/redirect',
```

Comment:
Why do we need walletWebURL here ? can't we use relative url ?

Reply:
This is the url where after auth rest needs to redirected so that we can handle auth resp request response 
validate hash and call gnap continue. Without relative path it goes to https://auth.rest.local/ relative not the wallet @Moopli

Reply:
Got it, thanks.